entries/r/restorecord.com.json

```diff
@@ -0,0 +1,13 @@
+{
+    "RestoreCord": {
+        "domain": "restorecord.com",
+        "url": "https://restorecord.com/login/",
```

Comment:
```suggestion
```

You don't need `url` if going to the base domain (e.g. `https://restorecord.com`) works.